# 1. CONCEPTOS BASICOS DE PROGRAMACION

## EJERCICIO 3
Escribe un programa en seudocódigo que solicite al usuario ingresar una temperatura en grados Celsius y luego la convierta a grados Fahrenheit utilizando la fórmula:
F=(C-9/5)+32
Donde:
• F es la temperatura en grados Fahrenheit.
• Ces la temperatura en grados Celsius.
Luego, muestra la temperatura en grados Fahrenheit.

In [12]:
"""
INICIO
    DEFINIR VARIABLE CELSIUS
    SOLICITAR GRADOS CELSIUS
    ASIGNAR VALOR A LA CELSIUS
    DEFINIR VARIABLE FAHRENHEIT
    CALCULAR FAHRENHEIT=(CELSIUS-9/5)+32
    ASIGNAR VALOR DE FAHRENHEIT
    IMPRIMIR RESULTADO
FIN
"""
C=float(input("Ingrese el valor numerico de la medida en grado celsius"))
F=(C-(9/5))+32
print(f"El valor en grados Fahrenheit es {F}")


El valor en grados Fahrenheit es 45.2


# 4 ENTRADA Y SALIDA DE DATOS

## EJERCICIO 3
Escribir un programa que pida al usuario su peso (en kg) y estatura (en metros), calcule el índice de masa corporal y lo almacene en una variable, y muestre por pantalla la frase Tu índice de masa corporal es donde es el índice de masa corporal calculado redondeado con dos decimales.

In [15]:
print("INGRESE PASA Y PESO")
masa=float(input("Ingrese su masa: "))
peso=float(input("Ingrese su peso: "))
imc=masa/(peso**2)
print(f"Su indice de masa corporal es: {imc}")

INGRESE PASA Y PESO
Su indice de masa corporal es: 22.03856749311295


# 5 FLUJO DE CONTROL

## EJERCICIO 1
Crear un programa que permita decidir a una persona cruzar la calle o no según:
• Si semáforo esta en verde cruzar la calle
• Si semáforo esta en rojo o amarillo no cruzar
La persona debe poder ingresar el estado del semáforo por teclado

## EJERCICIO 2
Escribir un programa que pregunte al usuario su edad y muestre por pantalla si es mayor de edad o no.

## EJERCICIO 3
Escribir un programa que almacene la cadena de caracteres contraseña en una variable, pregunte al usuario por la contraseña e imprima por pantalla si la contraseña introducida por el usuario coincide con la guardada en la variable sin tener en cuenta mayúsculas y minúsculas.

## EJERCICIO 4
Escribir un programa que pida al usuario un número entero y muestre por pantalla si es par o impar.

## EJERCICIO 5
Los tramos impositivos para la declaración de la renta en un determinado país son los siguientes:
"<table>\n",
    "    <thead>\n",
    "        <th style=\"text-align: center\">Renta</th>\n",
    "        <th style=\"text-align: center\">% de Impuesto</th>\n",
    "    </thead>\n",
    "    <tbody>\n",
    "        <tr>\n",
    "            <td style=\"text-align: center\">Menos de 10000€</td>\n",
    "            <td style=\"text-align: center\">5%</td>\n",
    "        </tr>\n",
    "        <tr>\n",
    "            <td style=\"text-align: center\">Entre 10000€ y 20000€</td>\n",
    "            <td style=\"text-align: center\">15%</td>\n",
    "        </tr>\n",
    "        <tr>\n",
    "            <td style=\"text-align: center\">Entre 20000€ y 35000€</td>\n",
    "            <td style=\"text-align: center\">20%</td>\n",
    "        </tr>\n",
    "        <tr>\n",
    "            <td style=\"text-align: center\">Entre 35000€ y 60000€</td>\n",
    "            <td style=\"text-align: center\">30%</td>\n",
    "        </tr>\n",
    "        <tr>\n",
    "            <td style=\"text-align: center\">Más de 60000€</td>\n",
    "            <td style=\"text-align: center\">45%</td>\n",
    "        </tr>\n",
    "    </tbody>\n",
    "</table>\n",
Realizar un programa que pueda decir el % de impuestos que una persona deba pagar según su sueldo

## EJERCICIO 6
Realiza un programa que lea dos números por teclado y permita elegir entre 3 opciones en un menú:
• Mostrar una suma de los dos números
• Mostrar una resta de los dos números (el primero menos el segundo)
• Mostrar una multiplicación de los dos números
• En caso de introducir una opción inválida, el programa informará de que no es correcta.

## EJERCICIO 7
La pizzería Bella Napoli ofrece pizzas vegetarianas y no vegetarianas a sus clientes. Los ingredientes para cada tipo de pizza aparecen a continuación.
•Ingredientes vegetarianos: Pimiento y tofu.
•Ingredientes no vegetarianos: Peperoni, Jamón y Salmón.
Escribir un programa que pregunte al usuario si quiere una pizza vegetariana o no, y en función de su respuesta le muestre un menú con los ingredientes disponibles para que elija. Solo se puede eligir un ingrediente además de la mozzarella y el tomate que están en todas la pizzas. Al final se debe mostrar por pantalla si la pizza elegida es vegetariana o no y todos los ingredientes que lleva.


## EJERCICIO 8
Escribí un programa que solicite al usuario una letra y, si es una vocal, muestre el mensaje "Es vocal". Verificar si el usuario ingresó un string de más de un carácter y, en ese caso, informarle que no se puede procesar el dato.